rclpy/executors/examples_rclpy_executors/custom_executor.py

```diff
@@ -46,7 +46,7 @@ class PriorityExecutor(Executor):
     def __init__(self):
         super().__init__()
         self.high_priority_nodes = set()
-        self.hp_executor = ThreadPoolExecutor(max_workers=multiprocessing.cpu_count())
+        self.hp_executor = ThreadPoolExecutor(max_workers=(os.cpu_count() or 4))
```

Comment:
Maybe remove the unnecessary parenthesis?

Reply:
53bd00f6a2a9a6ddb4961ae31d0a138d1bc52354